Library/include/CSP/Systems/Users/Authentication.h

```diff
@@ -97,6 +97,13 @@ class CSP_API LoginTokenInfo
 };
 
 
+enum ELoginStateResultFailureReason
+{
+	None,
```

Comment:
If `None` is an option for a failure reason then perhaps it should be instead 'Undefined' or "Unknown" ? If we get this on a client level, it would be interpreted as "it failed but there is no reason for the fail" which doesn't have enough info to explain itself...

Reply:
Would you be OK with an `Unknown` member with a value of -1 being added instead? `None` in this context means no `X-ErrorCode` header was present.

Reply:
Yes i think that's a bit better , thanks 🙂